great_expectations/data_context/store/__init__.py

```diff
@@ -15,6 +14,7 @@
 )
 from .database_store_backend import DatabaseStoreBackend  # isort:skip
 from .inline_store_backend import InlineStoreBackend  # isort:skip
+from .in_memory_store_backend import InMemoryStoreBackend  # isort:skip
```

Comment:
Any idea why we are isort: skipping all this stuff?

Reply:
I don't fully understand the purpose to be honest!

Reply:
oh wait I think we get issues if the base class isn't listed first